exec/java-exec/src/main/java/org/apache/drill/exec/physical/resultSet/impl/SingleVectorState.java

```diff
@@ -258,10 +258,12 @@ public int allocate(int cardinality) {
    * row, or for some previous row, depending on exactly when and where the
    * overflow occurs.
    *
-   * @param sourceStartIndex the index of the row that caused the overflow, the
+   * sourceStartIndex: the index of the row that caused the overflow, the
    * values of which should be copied to a new "look-ahead" vector. If the
    * vector is an array, then the overflowIndex is the position of the first
    * element to be moved, and multiple elements may need to move
+   *
+   * @param cardinality
```

Comment:
Would an accurate description for cardinality here be: the number of unique columns in the row?

Reply:
I think so.

Reply:
Okay, thank you. Should I also add descriptors to any empty Javadoc return statements?

Reply:
Yes please.